dist.ini

```diff
@@ -0,0 +1,26 @@
+name    = Email-Stuffer-TestLinks
+version = 0.01
+author  = binary.com <BINARY@cpan.org>
+license = Perl_5
+copyright_holder = binary.com
+copyright_year   = 2017
+
+[GatherDir]
+[MetaYAML]
+[License]
+[MakeMaker]
+[Manifest]
+[AutoPrereqs]
```

Comment:
Better to use `cpanfile`, since we have it in the repo anyway? Have a look at https://github.com/binary-com/perl-Date-Utility for an example.

Reply:
[AutoPrereqs] is now replaced by [Prereqs::FromCPANfile], hopefully I understood this right